game/plugin/skills/herblore/src/herblore.plugin.kts

```diff
@@ -0,0 +1,33 @@
+import org.apollo.game.message.impl.ItemOptionMessage
+import org.apollo.game.message.impl.ItemOnItemMessage
+
+on { ItemOptionMessage::class }
+        .where { option == 1 }
```

Comment:
Could you reduce this to 4 spaces instead of 8?  I think .editorconfig should be doing that (cc @Major- if it's not).

Reply:
(and same below)